tests/download.spec.ts

```diff
@@ -433,4 +443,35 @@ it.describe('download event', () => {
     expect(downloadPath).toBe(null);
     expect(saveError.message).toContain('File deleted upon browser context closure.');
   });
+
+  it('should be able to cancel pending downloads', async ({browser, server, browserName}) => {
+    it.skip(browserName !== 'chromium', 'Download cancellation currently implemented for only Chromium');
```

Comment:
we usually use `it.fixme` for the functionality that is going to be eventually implemented.